devel/ncurses/files/configure.diff

```diff
@@ -1,36 +0,0 @@
---- configure.orig	2015-08-05 19:20:32.000000000 +1000
-+++ configure	2015-08-17 16:48:10.000000000 +1000
-@@ -3166,7 +3166,7 @@
- program_transform_name=`echo $program_transform_name | sed -f conftest.sed`
- rm conftest.sed
- 
--for ac_prog in mawk gawk nawk awk
```

Comment:
@jmroot What do you think of the idea of adding a build-time dependency on a specific variant of awk and reintroducing a patch file to use only that one, in the interest of having a predictable build?